src/index.ts

```diff
@@ -1,21 +1,7 @@
-(function(self) {
-'use strict';
-
-function getModuleDefault(module) {
-  return module.default || module;
-}
-
-var appearsBrowserified = typeof self !== 'undefined' &&
```

Comment:
Is this a breaking change?

Reply:
It shouldn't break anyones app. The build step produces two files with unique consumption mechanisms that immediately inform whether you are *browserifying* or not depending on which one you choose to use. So, Pretender doesn't need to figure that out at runtime anymore.